public/locales/en/administration.json

```diff
@@ -113,5 +113,27 @@
         },
         "changeAdminError": "Unable to change admin status for ",
         "confirmChange": "Confirm Admin Status Change"
+    },
+    "sensors": {
+        "heading": "Current Sensors",
+        "name": "Sensor Name",
+        "purpleAirId": "Purple Air ID",
+        "latitude": "Latitude",
+        "longitude": "Longitude",
+        "status": "Current Status",
+        "validAqiTime": "Time of Most Recent Valid AQI",
```

Comment:
Consider adding a key above the table that explains more thoroughly the difference/nuance between last valid AQI (what is valid?) and last sensor reading. Seems like admins might forget this over time and not understand what it means for one vs the other, might be helpful to explain it in the admin page rather than having them find it somewhere in our clinic final report.

Reply:
Also info about what it means for a sensor to be active vs not

Reply:
I added a little question mark icon for more information, can you let me know your opinion on how it looks?